src/main/kotlin/no/nav/familie/ef/infotrygd/service/PeriodeService.kt

```diff
@@ -0,0 +1,26 @@
+package no.nav.familie.ef.infotrygd.service
+
+import no.nav.familie.ef.infotrygd.model.StønadType
+import no.nav.familie.ef.infotrygd.model.StønadType.BARNETILSYN
+import no.nav.familie.ef.infotrygd.repository.PeriodeRepository
+import no.nav.familie.ef.infotrygd.rest.api.Periode
+import no.nav.familie.ef.infotrygd.rest.api.PeriodeRequest
+import org.springframework.stereotype.Service
+
+@Service
+class PeriodeService(private val periodeRepository: PeriodeRepository) {
+
+    fun hentPerioder(request: PeriodeRequest): Map<StønadType, List<Periode>> {
+        val perioder = periodeRepository.hentPerioder(request).groupBy({ it.first }) { it.second }.toMutableMap()
+        perioder.put(BARNETILSYN, hentBarnetilsynPerioderMedBarn(perioder))
```

Comment:
Fint å fikse warnings direkte :) 
```suggestion
        perioder[BARNETILSYN] = hentBarnetilsynPerioderMedBarn(perioder)
```